pages/installation/coolify.mdx

```diff
@@ -3,21 +3,25 @@ title: Coolify
 description: How to install Postiz on Coolify
 ---
 
-import EarlyDoc from "../../components/snippets/earlydoc.mdx";
-import { Steps } from 'nextra/components';
-
-<EarlyDoc />
+import { Steps } from "nextra/components";
 
 <Steps>
 
+### Prerequisites
```

Comment:
This is good to have, but we should have the same steps for all methods. We have similar instructions on the docker compose page, so we should make this a snippet

Reply:
I understand, looking into it